.jscsrc.json

```diff
@@ -1,5 +1,6 @@
 {
   "preset": "google",
+  "esnext": true,
```

Comment:
would updating JSCS be a separate PR?
![](https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcRoWd3AyNYTyLJom3L2wyM3II3GTq6Wfy1CEdxRw6PmY5McYr8VIrhDoic)